### ISSUE TITLE
Increase contrast between nav menu text and background on hover

### ISSUE THREAD

**Issue by benlk** [INN/inn#109]
![Screen Shot 2019-05-21 at 22 04 59 ](https://user-images.githubusercontent.com/1754187/58142314-8e4b4b00-7c14-11e9-96ac-4521a718f1be.png)

The contrast ratio here is insufficient; #3b3b3b on #0089bb is a 2.82 contrast ratio. This should probably be #ffffff.
